.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You didn't want to add this file, do you?

Reply:
Nope, I don't even know where this is coming from, lol. Good catch!

Reply:
Alright since RST is pyhton docs primarly, my RST extension searches for its config. Fixed it by /dev/null it globally. Maybe something to consider later to mute py RST extensions.